src/locales/en.json

```diff
@@ -5,11 +5,81 @@
 // tab: This is the text of a navigation tab.
 
 {
+  // This is the name of a user Role.
   "role": {
-    // This is shown when a user does not have a role.
+    // This is shown when a user does not have a Role.
     "none": "None",
-    // This is the name of a user role.
-    "admin": "Administrator"
+    "admin": "Administrator",
+    "manager": "Project Manager",
+    "viewer": "Project Viewer"
+  },
+  "formState": {
+    // Form States control the lifecycle state of each Form. "Open" is one Form
+    // State.
+    "open": "Open",
+    // Form States control the lifecycle state of each Form. "Closing" is one
+    // Form State.
+    "closing": "Closing",
+    // Form States control the lifecycle state of each Form. "Closed" is one
+    // Form State.
+    "closed": "Closed"
+  },
+  "audit": {
+    // This is a category of actions performed on the server. It is shown when
+    // filtering the log of actions by type. Note that title case is used.
```

Comment:
This applies to the other comments below as well -- I don't know how helpful "Note that title case is used" is. Title case doesn't exist in most languages. It's unusual to have parentheses, though, and that might be worth calling out.

Reply:
Is this part of a drop down? That context would help pick the right words, I think.